Multisig/Data/Services/Utils/GSError.swift

```diff
@@ -313,6 +313,43 @@ enum GSError {
         let code = 1110
         let loggable = false
     }
+    
+    struct UDUnsuportedName: DetailedLocalizedError {
+        let description = "Can't use this name"
+        let reason = "Unstoppable name is invalid."
+        let howToFix = "Name should end with .crypto or .zil"
+        let domain = clientErrorDomain
+        let code = 1120
```

Comment:
Error codes should be aligned between iOS and Android. I did not check, are they?